frontend/src/ConfigManager.js

```diff
@@ -6,11 +7,12 @@ const DEFAULT_ID = 0;
 class ConfigManager {
   constructor() {
     this.graphID = DEFAULT_ID;
+    // eslint-disable-next-line global-require
     this.setConfig(require("./config.json"));
   }
 
   getGraphID = () => {
-    return this.graphID++;
+    return this.graphID + 1;
```

Comment:
This doesn't look equivalent to me. Are you sure we don't want to update `this.graphID` as well?

Reply:
Ah yes you're very right.